components/Chat/components/ChatLogin/ChatLogin.tsx

```diff
@@ -23,13 +23,15 @@ const ChatLogin = () => {
         Twitter
       </Button>
       <WalletConnectButton>
-        <Button
-          id="wallet_connect_btn_in_chat_box"
-          onClick={() => setUserType("wallet")}
-          className="!capitalize min-w-[150px] !bg-[black] !text-white !shadow-[0px_4px_4px_rgb(0,0,0,0.25)]"
-        >
-          Wallet
-        </Button>
+        {() => (
+          <Button
+            id="wallet_connect_btn_in_chat_box"
+            onClick={() => setUserType("wallet")}
+            className="!capitalize min-w-[150px] !bg-[black] !text-white !shadow-[0px_4px_4px_rgb(0,0,0,0.25)]"
+          >
+            Wallet
+          </Button>
+        )}
```

Comment:
did you change anything here?

just formatting changes right?

Reply:
right.